cmd/ansible-operator/main.go

```diff
@@ -26,7 +26,8 @@ import (
 
 func main() {
 	root := cobra.Command{
-		Use: "ansible-operator",
+		Short: "**NOTE** This executable is for use by operator-sdk and is not intended to be used manually",
```

Comment:
The binary can be run directly/manually, but only from within an Ansible project.
```suggestion
		Short: "Reconcile an Ansible operator project using ansible-runner",
		Long:  `This binary runs an Ansible operator that reconciles Kubernetes resources
managed by the ansible-runner program. It and can be run either directly or from an Ansible
operator project's image entrypoint
`,
```

Reply:
The root command has `run` and `version` subcommands (right?), so the root subcommand `Short` and `Long` should probably be something pretty general about interacting with ansible-based operators, maybe with a reference back to `operator-sdk`, like "Use `operator-sdk` to scaffold ansible-based operator projects".

The `run` subcommand should be more specific about actually loading `watches.yaml` and running controllers to reconcile ansible-operator APIs in a Kubernetes cluster. Not 100% on this, but the command _should_ be able to be executed from anywhere since there are flags to pass in the configuration.

Reply:
hi @jberkhahn, 

The binaries are used in the `make run` target scaffold in the Makefile. They allow users test/run the project outside of the cluster.

Reply:
If all ansible-/helm-operator's need is proper configuration and not an operator-sdk project (which I agree is the case), then we shouldn't mention operator-sdk in the help text @joelanford.